src/server/metrics.rs

```diff
@@ -27,14 +27,14 @@ lazy_static! {
             &["type"]
         ).unwrap();
 
-    pub static ref RECV_MSG_COUNTER: CounterVec =
+    pub static ref RECV_MSG_COUNTER_VEC: CounterVec =
```

Comment:
I personally don't like the naming style that attaches type name to the identifier.